src/sync/installation.ts

```diff
@@ -380,7 +392,7 @@ export async function maybeScheduleNextTask(
 		const delayMs = nextTaskDelaysMs.shift()!;
 		logger.info("Scheduling next job with a delay = " + delayMs);
 
-		await sqsQueues.backfill.sendMessage(jobData, (delayMs || 0) / 1000, logger);
+		await sqsQueues.backfill.sendMessage(jobData, Math.ceil((delayMs || 0) / 1000), logger);
```

Comment:
I also had to change the delay time here. Because sqsQueue has the delay in seconds, so we had to round it.